0014-managed-apps-transition-uvp/0014-managed-apps-transition-uvp.md

```diff
@@ -0,0 +1,30 @@
+# RFC 0014 - Giant Swarm Managed Apps UVP is the transition between DIY and Upstream Enterprise 
+
+Thu, 25. Feb 2021
+
+**What is a Unique Value Proposition (UVP)?** It is
+- (1) a distinctive value
+- (2) to a chosen set of customers
+- (3) at the right relative price.
+
+I realized sth while talking about Falco/Sysdig Secure in the SIG Partner Sync.
+
+**The strategy of all companies w open source products is to upsell (expensive) enterprise support. I wonder how much this strategy makes sense for customers?**
+
+To illustrate, one customer would have a few dozen apps (e.g. Kong, Aqua or Sysdig, Grafana Enterprise, Kafka, and I don’t know what else). **Doesn’t it seem too expensive and / or too inefficient to engage in enterprise contracts with each and every one of the companies behind each one?**
+
+I watched yesterday's [SIG Product Marketing Sync](https://gigantic.slack.com/archives/C95NTB55M/p1614169617017800?thread_ts=1614158491.014600&cid=C95NTB55M)
+
+Timo said,
+
+> Why partnering makes sense - We’re offering Loki, Prometheus, Grafana - we don’t see ourself as competitors. We see us as enablers of Grafana. But people DIY first. They do DIY w us first. Then they move on.
+
+Thinking from this perspective, questions for thought:
+
+- Could our ‘unique value proposition’ (UVP) be that we are a transition phase between DIY and upstream enterprise? Until they are sure they have more data that they **need** and **could benefit** from the value of upstream enterprise support?
+- If the above is true, how does that inform the depth of our management (scalability, performance, SLAs, etc.)?
+- If the above is true, how does that inform the depth of our support (level 1? level 2?)?
+- If the above is true, how does that inform the types of apps we should offer?
+- If the above is true, how does that inform our pricing?
+- Are we seeing the enterprise version of the consumer trend ‘subscription fatigue’?
```

Comment:
* Everything bundle for paid newsletters - $200 per year https://every.to/subscribe
* Curated Mac and iOS apps - $9.99 per month https://setapp.com/pricing